pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -53,32 +58,63 @@ func TestCertManager(t *testing.T) {
 		helm.Namespace(meshNamespace).
 			Chart("jetstack/cert-manager-istio-csr").
 			Release("istio-csr").
+			Version("v0.6.0").
 			ValuesString(istioCsrValues(meshNamespace, smcpName)).
 			Install(t)
 		oc.WaitPodsReady(t, meshNamespace, "app=cert-manager-istio-csr")
 
-		t.LogStep("Deploy the cert-manager in SMCP")
-		oc.ApplyString(t, meshNamespace, createSMCPWithCertManager(smcpName, meshNamespace, ns))
-		oc.WaitSMCPReady(t, meshNamespace, smcpName)
-
-		t.LogStep("Install httpbin and sleep")
-		app.InstallAndWaitReady(t, app.Httpbin(ns), app.Sleep(ns))
-
-		t.LogStep("Check if httpbin returns 200 OK ")
-		retry.UntilSuccess(t, func(t test.TestHelper) {
-			oc.Exec(t,
-				pod.MatchingSelector("app=sleep", ns),
-				"sleep",
-				`curl http://httpbin:8000/ip -s -o /dev/null -w "%{http_code}"`,
-				assert.OutputContains(
-					"200",
-					"Got expected 200 OK from httpbin",
-					"Expected 200 OK from httpbin, but got a different HTTP code"))
-		})
+		for _, ver := range []string{"v2.3", "v2.4"} {
+			t.NewSubTest(ver).Run(func(t test.TestHelper) {
+				t.Cleanup(func() {
+					app.Uninstall(t, app.Httpbin(ns.Foo), app.Sleep(ns.Foo))
+				})
+
+				t.LogStep("Deploy SMCP " + ver)
+				oc.ApplyString(t, meshNamespace, createSMCPWithCertManager(smcpName, meshNamespace, ns.Foo, ver))
+				oc.WaitSMCPReady(t, meshNamespace, smcpName)
+
+				t.LogStep("Verify that istio-ca-root-cert created in proper namespaces")
+				retryOpts := retry.Options().MaxAttempts(10).DelayBetweenAttempts(1 * time.Second)
```

Comment:
Why isn't the default 60 * 1 second good enough in this case?

The idea of using a default everywhere is to keep the code short and consistent.

Reply:
I didn't know this default setting. Removed.